app/controllers/admin_controller.rb

```diff
@@ -28,4 +29,12 @@ def technologies
   def claims
     @claims = ClaimRequest.all
   end
+
+  private
+
+  def authenticate_admin!
+    if  !user_signed_in? || !current_user.admin?
```

Comment:
It may read nice by writing this like `if !(user_signed_in? && current_user.admin?)` also make sure there is only one space after the `if`